terraform/production/main.tf

```diff
@@ -404,3 +404,21 @@ module "notifications" {
   alarm_sns_topic_emails    = local.all_notifications_email_addresses
   security_sns_topic_emails = local.all_notifications_email_addresses
 }
+
+#The auth alb is shared by keycloak and the auth service so we define the listener here and the rules in each repository
+resource "aws_lb_listener" "auth" {
+  load_balancer_arn = module.public_albs.keycloak.arn
```

Comment:
I've changed the name of the ALB module and output, I'll raise a PR to change Delta and the auth service.

I've left the CloudFront distribution still called keycloak, since that's technically still true while the auth service is internal only, and we can change it when we add public routes to the auth service public since we'll have to change the WAF then anyway.